runbotics/runbotics-scheduler/src/queue/processor/scheduler.processor.ts

```diff
@@ -172,4 +172,8 @@ export class SchedulerProcessor {
             );
         }
     }
+
+    private isIncorrectJobForProcessType(job: Job, process: IProcess) {
+        return (Boolean(job.data.input.variables) && !process.isAttended) || (!job.data.input.variables && process.isAttended);
+    }
```

Comment:
it won't stop stale form schedules. Use validation we have for process start endpoints